admin-console/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
+![](img/notifications-updates.png)
+- Updates for the cluster or for specific operators could also be listed here.
+- Clicking on one of these cards would take the user to a page with information about the updates.
+- Action buttons to install the update immediately could also be provided.
```

Comment:
I'm not sure how closely our use case will align, but I envision 20 operators having an update available...should we include an "Install all updates" eventually?

Reply:
yeah, that's a good idea. Is the console capable of  "handling" 20 simultaneous updates? or would it queue them or something?

Reply:
Great question :) @ tlwu2013 - Do you know the answer to this?